LayoutTests/editing/pasteboard/copy-with-shadow-tree-crash-live-range.html

```diff
@@ -17,7 +17,10 @@
   document.designMode = 'off';
   getSelection().modify('extend', 'left', 'lineboundary');
   getSelection().modify('extend', 'right', 'lineboundary');
-  getSelection().extend(div1);
+  try {
+      getSelection().extend(div1);
```

Comment:
I’m not sure that calling this and expecting it to fail by throwing an exception leaves this test in a helpful state. Since the crash is in the copy function why call this extend at all? Maybe we can remove the call. Or, if this extend was really needed, then are we really testing the crash any more or are we now just testing that the extend throws an exception, and calling copy in a way that would have not crashed?

Reply:
I guess we can delete the call. We're most certainly no longer testing whatever crash this test was created for. An alternative is to just delete the test and keep the one with LiveRangeSelectionEnabled=false.

Reply:
Added `<!-- webkit-test-runner [ LiveRangeSelectionEnabled=false ] -->` to this test instead.